docs/eigenda/overview.md

```diff
@@ -5,8 +5,8 @@ title: EigenDA Overview
 
 EigenDA is a data availability store made by
```

Comment:
Prior to mainnet launch, replace this entire sentence with the following blurb:
"EigenDA is a data availability system built by [EigenLabs](https://www.eigenlabs.org) and built on top of
[EigenLayer](../eigenlayer/overview/README.md). EigenDA is live on Holesky
[testnet](https://blobs-holesky.eigenda.xyz/) and will be available on Ethereum [mainnet](https://blobs.eigenda.xyz/) in early Q2 2024.

Reply:
Yes of course. This is not that PR though.